libs/model/src/token/GetTokens.query.ts

```diff
@@ -0,0 +1,61 @@
+import { type Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { QueryTypes } from 'sequelize';
+import { z } from 'zod';
+import { models } from '../database';
+
+export function GetTokens(): Query<typeof schemas.GetTokens> {
+  return {
+    ...schemas.GetTokens,
+    auth: [],
+    secure: false,
+    body: async ({ payload }) => {
+      const { search = '', cursor, limit, order_by, order_direction } = payload;
+
+      // pagination configuration
+      const direction = order_direction || 'DESC';
+      const order_col = order_by || 'name';
+      const offset = limit! * (cursor! - 1);
+      const replacements: {
+        search?: string;
+        offset: number;
+        order_col: string;
+        direction: string;
+        limit: number;
+      } = {
+        search: search ? `%${search.toLowerCase()}%` : '',
+        offset,
+        order_col,
+        direction,
+        limit,
+      };
+
+      const sql = `
+        SELECT *,
```

Comment:
We should explicitly select columns instead of selecting all, just in case there's ever some private info on the entity that we don't want to expose.